### PR TITLE
Fix:#4029 -  Add Release Notes to 1.30

### LINKED ISSUE #4029: Release Notes for Brave v1.30
- Implemented History Sync (#3227)
- Implemented Brave News to display inline ads (#3872)
- Added "Delete All" functionality to clear history (#3723)
- Added ability to share selected text from other apps to Brave via share sheet (#3863)
- Fixed an issue with rewards not being disabled when toggling off via rewards panel (#3591)
- Fixed app theme issue at launch, being set to light mode when Private Browsing only mode is enabled (#3936)
- Fixed menu to auto-close when a bookmark is selected (#3950)
- Fixed clear private data global settings from resetting to default when its changed (#3958)
- Fixed crash when editing/deleting favourites when focus is set on URL(show more mode) (#3974)

### PR DESCRIPTION
**Pull request by srirambv**
## Summary of Changes

This pull request fixes #4029

## Submitter Checklist:

- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`

## Test Plan:

NA

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [ ] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [ ] Adequate unit test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by iccub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.30](https://github.com/brave/brave-ios/releases/tag/v1.30)
+
+- Implemented History Sync. ([#3227](https://github.com/brave/brave-ios/issues/3227))
+- Implemented Brave News to display inline ads. ([#3872](https://github.com/brave/brave-ios/issues/3872))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: we can probably just use the same wording desktop did so it's the same across the board:

```
- Added display ads to the Brave News feed.
```

The one on desktop is a bit easier to understand re: not mentioning "inline".

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+- Added "Delete All" functionality to clear history via history page. ([#3723](https://github.com/brave/brave-ios/issues/3723))
+- Added ability to share selected text from other apps to Brave via share sheet. ([#3863](https://github.com/brave/brave-ios/issues/3863))
+- Fixed an issue with rewards not being disabled when toggling off via rewards panel. ([#3591](https://github.com/brave/brave-ios/issues/
```
Comment (kjozwiak) [inline CHANGELOG.md]: You're missing the issue # at the end of the link. Should be:

```
([#3591](https://github.com/brave/brave-ios/issues/3591))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+- Fixed app theme issue at launch, being set to light mode, when Private Browsing only mode is enabled. ([#3936](https://github.com/brave/brave-ios/issues/3936))
+- Fixed menu to auto-close when a bookmark is selected. ([#3950](https://github.com/brave/brave-ios/issues/3950))
+- Fixed "Clear private data" global settings from resetting to defaults when menu is closed after changing the settings. ([#3958](https://github.com/brave/brave-ios/issues/3958))
+- Fixed crash when editing/deleting favorites when focus is set on URL(show more mode). ([#3974](https://github.com/brave/brave-ios/issues/3974))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Space between `URL` & `(show more mode)`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.30](https://github.com/brave/brave-ios/releases/tag/v1.30)
+
+- Implemented History Sync. ([#3227](https://github.com/brave/brave-ios/issues/3227))
+- Implemented Brave News to display inline ads. ([#3872](https://github.com/brave/brave-ios/issues/3872))
+- Added "Delete All" functionality to clear history via history page. ([#3723](https://github.com/brave/brave-ios/issues/3723))
+- Added ability to share selected text from other apps to Brave via share sheet. ([#3863](https://github.com/brave/brave-ios/issues/3863))
+- Fixed an issue with rewards not being disabled when toggling off via rewards panel. ([#3591](https://github.com/brave/brave-ios/issues/
+- Fixed app theme issue at launch, being set to light mode, when Private Browsing only mode is enabled. ([#3936](https://github.com/brave/brave-ios/issues/3936))
```
Comment (kjozwiak) [inline CHANGELOG.md:10]: Can probably make this a bit easier to understand. Example:

```
- Fixed app theme being set to light at launch when "Private Browsing" only is being used as default.
```

**Review by kjozwiak — COMMENTED** [review]
Other than the few nits, everything else looks good 👍

File: CHANGELOG.md (hunk 3 above)
Comment (srirambv) [reply CHANGELOG.md]: Show more mode is when URL is focused so not adding an & there.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.30](https://github.com/brave/brave-ios/releases/tag/v1.30)
+
+- Implemented History Sync. ([#3227](https://github.com/brave/brave-ios/issues/3227))
+- Added display ads to the Brave News feed. ([#3872](https://github.com/brave/brave-ios/issues/3872))
+- Added "Delete All" functionality to clear history via history page. ([#3723](https://github.com/brave/brave-ios/issues/3723))
+- Added ability to share selected text from other apps to Brave via share sheet. ([#3863](https://github.com/brave/brave-ios/issues/3863))
+- Fixed app theme being set to light at launch when "Private Browsing" only is being used as default. ([#3591](https://github.com/brave/brave-ios/issues/3591))
+- Fixed app theme issue at launch, being set to light mode, when Private Browsing only mode is enabled. ([#3936](https://github.com/brave/brave-ios/issues/3936))
```
Comment (kjozwiak) [inline CHANGELOG.md:10]: Looks this should be removed as you have duplicate lines. Probably just forgot to more this one.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.30](https://github.com/brave/brave-ios/releases/tag/v1.30)
+
+- Implemented History Sync. ([#3227](https://github.com/brave/brave-ios/issues/3227))
+- Added display ads to the Brave News feed. ([#3872](https://github.com/brave/brave-ios/issues/3872))
+- Added "Delete All" functionality to clear history via history page. ([#3723](https://github.com/brave/brave-ios/issues/3723))
+- Added ability to share selected text from other apps to Brave via share sheet. ([#3863](https://github.com/brave/brave-ios/issues/3863))
+- Fixed app theme being set to light at launch when "Private Browsing" only is being used as default. ([#3591](https://github.com/brave/brave-ios/issues/3591))
```
Comment (kjozwiak) [inline CHANGELOG.md]: Wrong issue #. Should be `3936`. Probably a copy/paste error when adding the new line.

**Review by kjozwiak — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (srirambv) [reply CHANGELOG.md:10]: Thats a copy paste error. Tried with web and messed up. Fixed it

File: CHANGELOG.md (hunk 6 above)
Comment (srirambv) [reply CHANGELOG.md]: Wrong copy paste via web. Fixed it

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**